sys/stdio/stdio.c

```diff
@@ -61,6 +68,21 @@ void stdio_close(void) {
 #define MAYBE_WEAK __attribute__((weak))
 #endif
 
+ssize_t stdio_write_all(const void* buffer, size_t len)
```

Comment:
```suggestion
int stdio_write_all(const void* buffer, size_t len)
```

To me, `ssize_t` sends a lot of "this will return `len` on success vibes.

Reply:
Thanks for the vibe check, it's spot on, fixed.